### PR TITLE
Continuing code style cleanup

### PR DESCRIPTION
**Pull request by reillyeon**
Fixed errors E101, E111, E116, E123, E124, E126, E127 and E128.

### CODE REVIEW

File: presubmit.sh [hunk 1]
```diff
@@ -4,4 +4,4 @@
 set -e
 
 # Run flake8 to check coding style and consistency.
-flake8 --ignore=E101,E111,E116,E123,E124,E126,E127,E128,E201,E202,E203,E211,E221,E222,E225,E226,E227,E228,E231,E251,E261,E262,E265,E266,E402,E501
+flake8 --ignore=E116,E123,E124,E126,E127,E128,E201,E202,E203,E211,E221,E222,E225,E226,E227,E228,E231,E251,E261,E262,E265,E266,E402,E501
```
Comment (kb1lqc) [inline presubmit.sh]: Didn't you get rid of presubmit.sh with pytest?

File: Applications/APRS/aprs.py [hunk 2]
```diff
@@ -221,29 +221,24 @@ def sendPositions(stations, socket):
...
+                aprsPosition = ''.join([
+                    dataTypeIdent,
+                    latString,
+                    latitudeDirection,
+                    symbolTable,
+                    lonString,
+                    longitudeDir,
+                    symbol])
+
+                positionString = '{}>{},{},{}:{}.../{}/A={}{}\r'.format(
```
Comment (kb1lqc) [inline Applications/APRS/aprs.py:233]: Interesting use of a format(), very close to how `aprslib` does it. This code is going to be replaced by `aprslib` functions in the future FYI. I do want @kb1lqd to see this. Nifty formatting. WAYY cleaner than how I did it prior.

**Review by kb1lqc — CHANGES_REQUESTED** [review]
Please pull in the latest master to make sure this is still valid with the hermes and other code that has been modified. Looks good otherwise but I will need to check it with hardware prior to approving to ensure it doesn't break the software. I'm definitely cautious of the APRS changes and if they cause anything to break.

File: presubmit.sh (hunk 1 above)
Comment (binaryfox0) [inline presubmit.sh]: Separate pull request. I wasn't sure which order you'd get to them in.

Comment (reillyeon) [reply presubmit.sh]: I wasn't sure which order you'd get to these pull requests in so I based this one on master. GitHub doesn't seem to allow you to branch one pull request from another.

**Comment by kb1lqc** [comment]
looking good @reillyeon, Please fix the conflicting `presubmit.sh` and I'll merge!

**Comment by kb1lqc** [comment]
Awesome just saw these commits. @reillyeon are you planning on pushing more to this PR?

**Comment by reillyeon** [comment]
I have a few more errors I might get to before the night is done, but please merge this now.

**Comment by kb1lqc** [comment]
OK I think I'll get some stuff done on other Faraday items I was planning on doing tonight before moving over to this as I need to check it with hardware given the breadth of these changes.

File: Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py [hunk 3]
```diff
@@ -78,7 +78,8 @@ def CreateOutputFile(self):
             final_crc = hex(self.CalcTiTxtCrc16(self.section_data_index[i]))
 
             #Script Format
-            script_index_crc = "CRC_CHECK "+"0x"+str(final_addr)+' '+str(final_len)+' '+str(final_crc)
+            script_index_crc = "CRC_CHECK 0x{} {} {}".format(
```
Comment (kb1lqc) [inline Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py:81]: Looks benign but alerting @kb1lqd to some changes to the bootstrap loader. I'm tempted to pull this in without running it through firmware loading. You OK with remembering that this was touched if we encounter errors with bootstrap loading in the future?

**Review by kb1lqc — APPROVED** [review]

**Comment by kb1lqc** [comment]
Checked the following applications with hardware:
- Proxy
- Telemetry
- APRS
- Hermes
- Simpleconfig
- Readconfig
- Deviceconfig

I did not check the bootstrap loader but this may be acceptable. Thoughts @kb1lqd ?

File: Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py (hunk 3 above)
Comment (kb1lqd) [reply Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py:81]: I just testing @reillyeon branch and it fails. It hang's up. Master works fine.

image

File: Proxy/faraday_uart_stack/layer_2_protocol.py [hunk 4]
```diff
@@ -66,14 +66,14 @@ def rx_buffer_count(self):
...
-            time.sleep(0.001) #Shouldn't need this! BSALMI 6/13/16
+            time.sleep(0.001)  #Shouldn't need this! BSALMI 6/13/16
             if(self.enabled):
                 #Check for bytes to transmit over serial
                 if(not self.serial_tx_queue.empty()):
                     while(not self.serial_tx_queue.empty()):
                         self.ser.write(self.serial_tx_queue.get())
                 #Check for bytes to receive from serial
-                if((self.ser.inWaiting()>0)):
+                if self.ser.inWaiting() > 0:
```
Comment (kb1lqc) [inline Proxy/faraday_uart_stack/layer_2_protocol.py:76]: Seems benign but also suspicious

**Review by kb1lqc — CHANGES_REQUESTED** [review]
Need to figure out why BSL isn't working right.

**Comment by reillyeon** [comment]
As far as I can tell my branch is generating the same `FaradayFirmwareUpgradeScript.txt`. It's the actual firmware update tool that is crashing in your screenshot. Very odd indeed.

**Comment by kb1lqc** [comment]
@reillyeon and @kb1lqd This appears to work fine for me...
bootstrap

File: Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py (hunk 3 above)
Comment (kb1lqd) [reply Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py:81]: OK So it was a folder permission or something on my computer. I copied all of the BSL folder contents into my master and ran it with success as well.

I made a fake firmware version with "5555" for firmware to verify firmware loading.

image

**Review by kb1lqc — APPROVED** [review]

**Comment by kb1lqc** [comment]
@kb1lqd confirmed this works with the Bootstrap Loader. He has some file permission error on his computer that was cleared by changing directories. Merging.

**Pull request merged**